packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -193,21 +193,25 @@ export function ArticleEditor({id}: ArticleEditorProps) {
   }
 
   // Reads title and lead from the first block and saves them in variables
-  function syncBlocksWithMeta() {
-    const titleBlock = blocks.map(block => {
-      if (block.type === 'title') {
-        return block.value
-      } else {
-        return false
-      }
-    })
+  function syncFirstTitleBlockWithMetadata() {
+    interface titleBlockToSync {
```

Comment:
That interface already exists in `packages/editor/src/client/blocks/types.ts` as `TitleBlockValue`. And internfaces start in general with a capital letter.